src/app/features/character-panel/character-panel.component.ts

```diff
@@ -67,17 +68,280 @@ export class CharacterPanelComponent implements OnInit {
     this.mirageService.mirageData$.subscribe(
       ({ character: mirageCharacter }) => {
         this.character = mirageCharacter;
+        this.refreshItemList();
+
       }
     );
 
     this.armageddon.characterData$.subscribe((data) => {
       this.zone.run(() => {
         this.character = data;
+        this.refreshItemList();
       });
     });
   }
 
-  getSegmentedHungerValue() {
+  // Character Card utilities
+  itemList = [
```

Comment:
we need an interface for these ones

Reply:
Will come back to this in a future change; I have a Class planned

Reply:
let's give it a temporary interface in the meantime just so we all know what to put there and don't need to analyze the code too much